prisma/seed.ts

```diff
@@ -25,6 +25,31 @@ async function main() {
       name: "Keith Full Auth Flow",
     },
   });
+  await prisma.category.create({ data: { name: "Sea Shanties" } });
+  await prisma.category.create({ data: { name: "New American Americana" } });
+  await prisma.category.create({ data: { name: "Folk" } });
+  await prisma.category.create({ data: { name: "Protest" } });
+  await prisma.category.create({
+    data: { name: "Really makes you go hmmmmm." },
+  });
+  await prisma.category.create({
+    data: { name: "Really makes you go hummmmm." },
+  });
+
+  await prisma.song.create({
+    data: {
+      id: 1,
+      createdAt: "2023-01-26T12:41:28.070Z",
+      updatedAt: "2023-01-26T12:41:56.718Z",
+      title: "test",
+      attribution: "te",
+      stanzas:
```

Comment:
Can we use the `createMockStanza()` function here instead of hard coding the string in the seeds?